### DISCUSSION TITLE
Using objection to get PCAP from ssl transport

### DISCUSSION THREAD

**Discussion by joko99999** [sensepost/objection#691] [Q&A]
Can I use objection to create a PCAP file that contains ssl_read and ssl_write decrypted?

**Comment by IPMegladon** [comment]
This would likely be easier to do with a Frida script, I'm sure there are already ones out there that do exactly this. Suppose you could also just use tcpdump if you are on a rooted/jailbroken device though?

**Comment by joko99999** [comment]
Sure. The possible with Frida, but with objection that more comfortable for usage.
Maybe that not hard to integrate 
https://github.com/saleemrashid/frida-sslkeylog

**Comment by IPMegladon** [comment]
Objection doesnt currently have the functionality, but I'm sure you are welcome to make a PR for it. Alternatively you can also implement it as plugin: https://github.com/sensepost/objection/wiki/Plugins
